utils/oscap-cvrf.c

```diff
@@ -72,7 +72,7 @@ static int app_cvrf_evaluate(const struct oscap_action *action)
 		if (oscap_err())
 			fprintf(stderr, "%s %s\n", OSCAP_ERR_MSG, oscap_err_desc());
 
-	oscap_free(action->cvrf_action);
+	free(action->cvrf_action);
```

Comment:
Yes, it was like this before but I briefly changed it, then realized it wasn't working with oscap_free